yt_dlp/extractor/mediaklikk.py

```diff
@@ -143,8 +143,7 @@ def _real_extract(self, url):
         if not playlist_url:
             raise ExtractorError('Unable to extract playlist url')
 
-        formats = self._extract_wowza_formats(
-            playlist_url, video_id, skip_protocols=['f4m', 'smil', 'dash'])
+        formats = self._extract_m3u8_formats(playlist_url, video_id)
```

Comment:
we could do this pre-emptively
```suggestion
        formats, subtitles = self._extract_m3u8_formats_and_subtitles(playlist_url, video_id)
```
and then make sure to return `subtitles` below

Reply:
Extracted and returned subtitles as well.